core/src/main/java/site/ycsb/measurements/OneMeasurementHdrHistogram.java

```diff
@@ -142,26 +153,33 @@ public void exportMeasurements(MeasurementsExporter exporter) throws IOException
 
   /**
    * This is called periodically from the StatusThread. There's a single
-   * StatusThread per Client process. We optionally serialize the interval to
-   * log on this opportunity.
+   * StatusThread per Client process. We optionally serialize the interval to log
+   * on this opportunity.
+   * 
+   * @throws IOException
    *
    * @see site.ycsb.measurements.OneMeasurement#getSummary()
    */
   @Override
   public String getSummary() {
+    String buckets = "";
+
     Histogram intervalHistogram = getIntervalHistogramAndAccumulate();
     // we use the summary interval as the histogram file interval.
     if (histogramLogWriter != null) {
       histogramLogWriter.outputIntervalHistogram(intervalHistogram);
     }
+    if (addstatus) {
+      buckets += getStatusCounts();
```

Comment:
why is this doing concatenation instead of assignment?

Reply:
good question, I guess out of habit (and mostly me doing python :))
fixed